frontend/public/components/_dropdown.scss

```diff
@@ -35,6 +35,10 @@ $color-bookmarker: #DDD;
 
 .dropdown-menu__filter {
   padding: 5px 10px 10px;
+
+  .dropdown--text-filter {
+    float: none;
```

Comment:
Why is this style needed?

Reply:
Without it, the input's wrapper is floating left which causes the first option below it to display very strangely. It is a style that is coming from a bootstrap _button-groups.scss.
![screen shot 2018-08-15 at 8 08 58 am](https://user-images.githubusercontent.com/26305082/44147540-88cc3e66-a062-11e8-8ea7-7f9d9b4a861d.png)
![screen shot 2018-08-15 at 8 09 06 am](https://user-images.githubusercontent.com/26305082/44147544-8a18db08-a062-11e8-92e4-8eb5ddb04544.png)

Reply:
OK I see.

The rule shouldn't need to be nested since it has BEM naming. Can you remove the nesting?

I would add a comment this is specifically needed for use inside Bootstrap button groups and double check that it doesn't cause problems in other places we use typeahead.